src/ipUtils.c

```diff
@@ -141,7 +141,8 @@ void inet_ntop6 (const uint8_t *src, char *dst, size_t dstSize)
 			tp += strlen(tp);
 			break;
 		}
-		ASSERT((unsigned int) i < SIZEOF(words));
+		STATIC_ASSERT(sizeof(unsigned) >= sizeof(words[i]), "oversized type for %u");
```

Comment:
I think we are using `SIZEOF` instead of `sizeof` for variables (adds a pointer vs. array check, does not work for some specific array size, 4 or something).

Reply:
I thought about that, but for printf things it doesn't really matter, you never print an array unless it's %s. Can replace it if the aesthetics would be better, no problem.

Reply:
Ah, SIZEOF doesn't really work for 4 byte types, since it's the same size as a pointer. Interesting, though it's smarter than that.
`(sizeof(__typeof(int[0 - (sizeof(var) == sizeof((void *)0))])) * 0)`